internal/sql/driver/statement.go

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package driver
+
+import (
+	"context"
+	"database/sql/driver"
+)
+
+type Statement struct {
+	ss    *SQLService
+	query string
+}
+
+func newStatement(query string, ss *SQLService) *Statement {
+	return &Statement{
+		query: query,
+		ss:    ss,
+	}
+}
+
+func (s Statement) Close() error {
+	return nil
+}
+
+func (s Statement) NumInput() int {
+	return -1
+}
+
+func (s *Statement) Exec(args []driver.Value) (driver.Result, error) {
```

Comment:
Can you also check [StmtExecContext](https://pkg.go.dev/database/sql/driver#StmtExecContext) and [StmtQueryContext](https://pkg.go.dev/database/sql/driver#StmtQueryContext)? 🙏
 
Because It looks like we have everything ready for context support 🚀

Reply:
Thanks! I've added them at: https://github.com/hazelcast/hazelcast-go-client/pull/695/commits/c63fe8e2987fe317b8b18296dc3adeb6043e8616